Add publishing a form to a page instruction to quickstart page (#5)
AlexaCRM/alexacrm.github.io: wpcrm/quickstart.md

![Create a new form using Solution Explorer](/img/wpcrm/createform-solutionexplorer.png)

In Solution Explorer navigate to **Components > Entities** section and choose the desired entity, for example, *Contact*. Expand the selected entity node and select *Forms*. Click the *New* button and select the desired form type.
For more information about form types please see [Create and design forms](https://docs.microsoft.com/en-us/dynamics365/customer-engagement/customize/create-design-forms).

![Form editor](/img/wpcrm/createform-editor.png)

Now you can create and save the new form using the form editor.
For more information about the form editor please see [Form editor documentation](https://docs.microsoft.com/en-us/dynamics365/customer-engagement/customize/form-editor-user-interface-legacy).

> Please don't forget to specify name for your custom form and publish it.

Reviewer: ```suggestion
Please don't forget to specify the name of your custom form and publish it.
```

### Adding a form to a page

Create a new page in your WordPress administration panel by navigating to **Pages &gt; Add new** or open an existing one from **Pages &gt; All pages** menu section. Don't forget to enter page title if it's a new page.

> Switch content editor mode to plain text by clicking on the *Text* tab to be able to insert code in content field. Finally insert this code to *Content* text field:

Reviewer: ```suggestion
Switch the content editor mode to plain text by clicking on the *Text* tab. Finally insert the following code into the editor:
```

{% raw %}
```twig
[msdyncrm_twig]
{% form entity="contact" name="WordPress Example Form" mode="create" %}{% endform %}
[/msdyncrm_twig]
```
{% endraw %}

> Where

Reviewer: Please remove this "where" section. The parameters used here are very self-evident. Maybe leave the `mode`. Say something like "In `create` mode the form will create a new record in CRM".

- `entity` - the entity name (in this case it is *contact*)
- `name` - the name of the form associated with the entity
- `mode` - form operation mode (in this case we want to add new contact record so mode set to *create*)

![Dynamics 365 Advanced Find window.](/img/wpcrm/quickstart-form-add.png)

Click *Publish* button on the right sidebar to publish your new page or *Update* button if you are editing an existing page.

> Now you should see your form on site page and should be able to add new contacts to Dynamics 365 by submitting this form.

Reviewer: ```suggestion
Now you should see your form on a website page. This form will submit new contacts directly to Dynamics 365.
```
